File: 2. Managers/MusicAPI.Managers/ViewModels/Artists.cs
namespace MusicAPI.Managers.ViewModels
{
    public class Artists
    {
        public string Href { get; set; } = string.Empty;

        public int Limit { get; set; } = 0;

        public string Next { get; set; } = string.Empty;

        public int Offset { get; set; } = 0;

        public string Previous { get; set; } = string.Empty;

        public int Total { get; set; } = 0;

        public Artist[] Items { get; set; } = Array.Empty<Artist>();
    }
}

File: 1. Clients/angular-showcase/src/app/modules/data-models/artists.ts
import { Artist } from "./artist";

export class Artists {
  constructor() {
    this.href = '';
    this.limit = 0;
    this.next = '';
    this.offset = 0;
    this.previous = '';
    this.total = 0;
    this.items = [];
  }

  href: string;
  limit: number;
  next: string;
  offset: number;
  previous: string;
  total: number;
  items: Artist[];
}
